docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,131 @@
+Django <installing-official-release>`.
+
+   .. console::
+
+        $ python -m pip install django-debug-toolbar
+
+.. admonition:: Managing application dependencies
+
+    Managing dependencies is an important part developing an application, but
+    it is outside of the scope of this tutorial. Please see the official Python
+    documentation regarding `Managing Application Dependencies
+    <https://packaging.python.org/en/latest/tutorials/managing-dependencies/>`_
+    for more information.
+
+Third-party packages that integrate with Django may need some setup.
+Sometimes you will need to add the package's Django app to your
+:setting:`INSTALLED_APPS` setting. `The Django Debug Toolbar has many
+installation steps
+<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
+We're linking to the toolbar's documentation here because as a third-party
+package, it's maintained outside of the core development of Django. This
+allows the package maintainers to make releases outside of Django's release
+schedule
+
+Once installed, you should be able to see the DjDT "handle" on the right side
+of the browser window when you refresh the polls application.
+
+Getting help from others
+========================
+
+At some point you will run into a problem, for example the
+toolbar may not render. When this happens and you're unable to
+resolve the issue yourself, there are options available to you.
+
+#. If the problem is with a specific package, check if there's a
+   troubleshooting of FAQ in the package's documentation. For example the
+   Django Debug Toolbar has a `Tips section
+   <https://django-debug-toolbar.readthedocs.io/en/latest/tips.html>`_ that
+   outlines troubleshooting options.
+#. Consult the `Django Forum <https://forum.djangoproject.com/>`_.
+#. Join the `Django Discord server <https://discord.gg/xcRH6mN4fa>`_.
+#. Join the #Django IRC channel on `Libera.chat <https://libera.chat/>`_.
+
+Installing other third-party packages
+=====================================
+
+Another popular third-party package is `Django REST framework
+<https://www.django-rest-framework.org/>`_, also known as "DRF". This is a
+great package to use if you want to use your Django application in a headless
+design where Django only exposes an API while the frontend of your application
+is handled either in a mobile application or a JavaScript application.
+
+There are many more third-party packages than the Django Debug Toolbar and
+Django REST framework. You can find more third-party packages using the
+fantastic Django resource, `Django Packages <https://djangopackages.org/>`_.
+
+- Here's a `grid of development tool packages
+  <https://djangopackages.org/grids/g/developer-tools/>`_.
+- Here's a `grid of packages related to creating APIs
+  <https://djangopackages.org/grids/g/api/>`_.
+
+Evaluating third-party packages
+===============================
+
+It can be difficult to know what third-party packages you should use. This
```

Comment:
perhaps could insert a reference to the data that django packages shows? it can help evaluate.

Reply:
I'm unsure what you mean. By reference do you mean a line describing a grid from django packages?

Reply:
@adamchainz Do you have a specific edit here? (I'd have just headed to your post given this paragraph...) 🤔

Reply:
see below